### PR TITLE
update the backend

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Update Backend](https://trello.com/c/uoLqllcf/859-update-backend)

File: datatypes/src/collections/batch_builder.rs [hunk 1]
```diff
@@ -328,7 +328,7 @@ impl RawFeatureCollectionBuilder {
             .add_buffer(values_buffer);
 
         let data = if let Some(nulls) = nulls_buffer {
-            builder.null_bit_buffer(nulls).build()
+            builder.null_bit_buffer(Some(nulls)).build()
```
Comment (jdroenner) [inline datatypes/src/collections/batch_builder.rs:331]: the "nulls" buffer is already an option. Can't we use that? Maybe the if else block is obsolete now?

File: datatypes/src/collections/feature_collection.rs [hunk 2]
```diff
@@ -1646,10 +1646,10 @@ mod tests {
             empty_hash_map_size
         );
 
-        let struct_stack_size = 144;
+        let struct_stack_size = 168;
```
Comment (jdroenner) [inline datatypes/src/collections/feature_collection.rs:1649]: why did this change?

File: datatypes/Cargo.toml [hunk 3]
```diff
@@ -14,20 +14,20 @@ postgres = ["postgres-types", "postgres-protocol"]
...
+geo = "0.21"
+geojson = "0.23"
 image = "0.24"
 num = "0.4"
 num-traits = "0.2"
 ordered-float = { version= "3.0", features = ["serde"] }
 paste = "1.0"
 postgres-protocol = { version = "0.6", optional = true }
-postgres-types = { version = "0.2", features = ["derive", "with-chrono-0_4", "with-uuid-0_8"], optional = true }
+postgres-types = { version = "0.2", features = ["derive", "with-chrono-0_4", "with-uuid-0_8", "with-uuid-1"], optional = true }
```
Comment (jdroenner) [inline datatypes/Cargo.toml:30]: why do we need both?

File: datatypes/src/collections/feature_collection.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/collections/feature_collection.rs:1649]: Maybe they added a field to the struct or changed some internals :shrug: .

File: datatypes/Cargo.toml (hunk 3 above)
Comment (ChristianBeilschmidt) [reply datatypes/Cargo.toml]: not yet, waiting for bb8: https://github.com/djc/bb8/issues/127

reverted it here:

[981f8ad](https://github.com/geo-engine/geoengine/pull/556/commits/981f8adf646d900b357a28eca4c49a831b459453)

File: datatypes/src/collections/batch_builder.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/collections/batch_builder.rs]: good catch!
[a0faed2](https://github.com/geo-engine/geoengine/pull/556/commits/a0faed2feeee14341533dc5edd8282dc612319f0)

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**